src/explosion.cpp

```diff
@@ -442,7 +442,7 @@ std::vector<tripoint> game::shrapnel( const tripoint &src, int power,
     // Basically bisect between 0 and map diameter using shrapnel_calc().
     // Need to update shadowcasting to support limiting range without adjusting initial distance.
     const tripoint start = { 0, 0, src.z };
-    const tripoint end = { m.getmapsize() *SEEX, m.getmapsize() *SEEY, src.z };
+    const tripoint end = { m.getmapsize() * SEEX, m.getmapsize() * SEEY, src.z };
```

Comment:
`astyle` v3.0.1 thinks you're dereferencing the pointers `SEEX` and `SEEY`, thus why it wants
```suggestion
    const tripoint end = { m.getmapsize() *SEEX, m.getmapsize() *SEEY, src.z };
```

...should I grab a newer astyle for the build machine?

Reply:
No let's not open that cannon worms, this was just me manually "fixing" it and forgetting to make sure astyle agreed with me.